### PR TITLE
Support protoc plugin generated files.

### PR DESCRIPTION
**Pull request by luluz66**
protoc plugins can generate additional .pb.go files in the same
directory as the standard .pb.go.This PR allows us to copy these
additional .pb.go files too.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -116,29 +116,42 @@ type languageProtoRule struct {
...
 	case goProtoLibrary:
 		wsRelpath := githubRepoRe.ReplaceAllLiteralString(r.importPath, "")
 		if wsRelpath == r.importPath {
-			return "", "", fmt.Errorf("could not figure out workspace relative path for import %q", r.importPath)
+			return []srcAndDest{}, fmt.Errorf("could not figure out workspace relative path for import %q", r.importPath)
+		}
+		srcDir := filepath.Join(bazelBin, filepath.Dir(protoRelpath), r.name+"_", r.importPath)
+		srcs, err := filepath.Glob(srcDir + "/*.pb.go")
+		if err != nil {
+			return []srcAndDest{}, nil
```
Comment (bduffany) [inline main.go]: nit: return `nil, err` here instead?

File: main.go [hunk 2]
```diff
@@ -116,29 +116,42 @@ type languageProtoRule struct {
...
 
 	case tsProtoLibrary:
 		src := filepath.Join(bazelBin, filepath.Dir(protoRelpath), r.name+".d.ts")
 		dest := filepath.Join(workspaceRoot, filepath.Dir(protoRelpath), r.name+".d.ts")
-		return src, dest, nil
+		return []srcAndDest{{src: src, dest: dest}}, nil
 
 	}
-	return "", "", fmt.Errorf("unknown proto rule kind %q", r.kind)
+	return []srcAndDest{}, fmt.Errorf("unknown proto rule kind %q", r.kind)
```
Comment (bduffany) [inline main.go]:
```suggestion
	return nil, fmt.Errorf("unknown proto rule kind %q", r.kind)
```

**Review by bduffany — APPROVED** [review]

File: main.go (hunk 2 above)
Comment (luluz66) [reply main.go]: Done

File: main.go (hunk 1 above)
Comment (luluz66) [reply main.go]: Done

**Pull request merged**